src/client/components/app/Transactions/components/TxContainer.tsx

```diff
@@ -62,7 +62,6 @@ const StyledTxContainer = styled.div`
   flex-direction: column;
   padding: ${({ theme }) => theme.card.padding};
   width: 100%;
-  height: 53rem;
```

Comment:
This will cause rerendering and wrong style in most of the txcontainers, we should leave it or change the height to another value to reflect the new changes and don't cause an overflow

Reply:
Height should be defined by the content and not the txcontainer (maybe we can set a max one), which of the other contents would make this break?

Reply:
In the designs the height was fixed, and it was not defined by the content in all cases. More like having a max and min height. If not, it will overflow in the vault details, or cause rerenders

Reply:
<img width="679" alt="image" src="https://user-images.githubusercontent.com/78362532/164791123-a5d7d3f6-b91c-4d9f-9061-5fa57b709d39.png">
In this case, for exxample, we should be flexing the content to the bottom to occupy all the remaining space